real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/GpsPanel.java

```diff
@@ -42,6 +42,7 @@ public double getLongitude(){
 	private int frameWidth;
 	private int frameHeight;
 	private Subscriber gpsSub;
+	private Subscriber encSub;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1743642523/issues/source?bid=2938665&fileBranchId=3058909#l45)